go/consensus/tendermint/abci/mux.go

```diff
@@ -858,6 +862,10 @@ func (mux *abciMux) checkDependencies() error {
 	return nil
 }
 
+func (mux *abciMux) MockClose() {
+	mux.doCleanup()
```

Comment:
this should probably be behind the debug flag?

Reply:
actually can these methods be removed from `abciMux` struct, and a `mockAbciMux` struct is created in the `mux_mock.go` that embeeds `abciMux` and adds these two methods?